### PR TITLE
fix duplicate webhook insert operation

### PR DESCRIPTION
**Pull request by gy95**
#### What type of PR is this?
/kind cleanup

#### What this PR does / why we need it:
when webook is duplicate, we don't need to add webook to the set once again, So just add `continue` statement to skip the `Insert` operation and to continue to the next loop.
#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

**Comment by gy95** [comment]
kindly ping @mikedanese
PTAL

**Comment by gy95** [comment]
could you please take a look? @thockin
thanks 😄

**Comment by gy95** [comment]
ping @kubernetes/api-approvers

File: pkg/apis/admissionregistration/validation/validation.go [hunk 1]
```diff
@@ -248,6 +249,7 @@ func validateMutatingWebhookConfiguration(e *admissionregistration.MutatingWebho
 		if opts.requireUniqueWebhookNames && len(hook.Name) > 0 {
 			if hookNames.Has(hook.Name) {
 				allErrors = append(allErrors, field.Duplicate(field.NewPath("webhooks").Index(i).Child("name"), hook.Name))
+				continue
 			}
 			hookNames.Insert(hook.Name)
```
Comment (liggitt) [inline pkg/apis/admissionregistration/validation/validation.go:254]: putting the `Insert(...)` in an `else` block seems better than a `continue`. The `continue` is correct today but could introduce a bug in the future if more validation was added after the `if opts.requireUniqueWebhookNames` block

same comment below

Comment (gy95) [reply pkg/apis/admissionregistration/validation/validation.go]: done, please check again, thanks 😄

**Pull request merged**